### PR TITLE
Clarify the installation instructions for issue # 1120 vispy/vispy master

### PR DESCRIPTION
**Pull request by 00krishna**
@rossant requested a PR submission to change the vispy install instructions to include:
- install miniconda/anaconda
- conda install numpy vispy pyqt
- install the latest proprietary GPU drivers
- run some of the examples
- note that the test suite might be unstable on some systems, which is somewhat unrelated to the working state of the examples

I added these instructions + explanation to the installation.rst file within my personal fork of the  vispy/vispy-website repository. The commit message is below.

Fix installation.rst. Added detail to installation options section ba…sed upon recommendations from issue #1120 by @rossant.

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -44,6 +44,14 @@ your system and video driver. The OpenGL version must be at least 2.1.
 Installation options
 ====================
 
+**Before installing VisPy** you should ensure a working version of python is installed on your computer, including all of the requirements included in the **Backend Requirements** section above. A simple way to install most of these requirements is to install the **Anaconda** scientific python distribution from Continuum Analytics. [Anaconda](https://www.continuum.io/downloads) will install most of the VisPy dependencies for you. If yor computer is low on hard disk space, or you would like a minimal python installation, you may install the [Miniconda](http://conda.pydata.org/miniconda.html) package also from Continuum Analytics. Once Anaconda is installed, create a [conda python environment](http://conda.pydata.org/docs/py2or3.html).
+
```
Comment (00krishna) [inline installation.rst:48]: instructions to install miniconda/anaconda

File: installation.rst [hunk 2]
```diff
@@ -44,6 +44,14 @@ your system and video driver. The OpenGL version must be at least 2.1.
...
+Next, install the following VisPy dependencies directly through `pip` or the Anaconda package installer.
+
+$ conda install numpy pyqt
+
```
Comment (00krishna) [inline installation.rst:52]: instructions to install numpy and pyqt through conda package manager. Note I also mentioned that the user should create a conda environment first--and also included a link to the Anaconda documentation on how to do this.

File: installation.rst [hunk 3]
```diff
@@ -44,6 +44,14 @@ your system and video driver. The OpenGL version must be at least 2.1.
...
+Once the python dependencies have been installed, install the latest proprietary drivers for your computer's GPU. Generally these drivers may be downloaded from the GPU manufacturer's website.
+
```
Comment (00krishna) [inline installation.rst:54]: Instructions for install latest GPU drivers.

File: installation.rst [hunk 4]
```diff
@@ -71,3 +79,5 @@ check if everything is ok. To do this, just type::
    >>> import vispy
    >>> vispy.test()
    ...
+
+Please note that the test suite may be unstable on some systems. Any potential instability in the test suite does not necessarily imply instability in the working state of the provided VisPy examples.
```
Comment (00krishna) [inline installation.rst:83]: Additional comment about the test suite instability.

File: installation.rst (hunk 1 above)
Comment (rossant) [inline installation.rst:48]: - do these Markdown-formatted links work in ReST documents?
- typo in `If yor computer`

**Comment by rossant** [comment]
thanks @00krishna ! LGTM except minor comment
@Eric89GXL feel free to merge once the comments have been taken into account

**Comment by djhoese** [comment]
This has been merged through #9

**Pull request closed without merging**